src/cgame/cg_view.cpp

```diff
@@ -1551,8 +1551,8 @@ static void CG_AddReverbEffects( vec3_t loc )
 		}
 
 		dist = CM_DistanceToModel( loc, cgs.gameReverbModels[i] );
-		weight = 1.0f - dist / cgs.gameReverbDistances[i];
-		weight = Math::Clamp( weight, 0.0f, 1.0f ); // Maths::clampFraction( weight )
+
+		weight = 1.0f - dist / std::max( 0.01f, cgs.gameReverbDistances[i] );
```

Comment:
I rewrote it in the same ways it has been done for `cgs.gameGradingDistances` in:

- https://github.com/Unvanquished/Unvanquished/pull/3213